src/app/views/discover/games/view/view.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="route-discover-games-view" v-if="game">
+	<div class="route-discover-games-view" v-if="game && game.developer">
```

Comment:
Why this check? when would it not have a developer?

Reply:
Removed.